### PR TITLE
Display full name for test cases [fixes #17]

### LINKED ISSUE #17: Add class name to test cases

### PR DESCRIPTION
**Pull request by baev**
screen shot 2016-11-21 at 18 40 02

### CODE REVIEW

**Comment by baev** [comment]
Summon @eroshenkoam @just-boris

**Comment by just-boris** [comment]
This line is going to jump when you are switching between cases that have a full name to cases, that have not. Let's reserve some height for this block, or put it bellow the usual name.

**Comment by baev** [comment]
> This line is going to jump when you are switching between cases that have a full name to cases

Is that a problem? Usually all the test cases have a full name, so everything should be fine. Some extra reserved space looks bad imho. 

Probably we should aways use only one line and hide the rest of it

**Comment by just-boris** [comment]
> Is that a problem? 
I was getting some complaints about that in Allure 1 and did some work to improve it.
Here the story comes again.

Of course we can postpone the fix, until somebody else will report it.

File: face/src/components/testcase/TestcaseView.hbs [hunk 1]
```diff
@@ -1,4 +1,7 @@
 <a class="pane__overlay" href="#{{route.baseUrl}}/{{uid}}"></a>
+{{#if fullName}}
+    <div class="pane__subtitle">{{fullName}}</div>
+{{/if}}
```
Comment (just-boris) [inline face/src/components/testcase/TestcaseView.hbs]: empty div has zero height, you can safely get rid of #if to make it cleaner

**Comment by baev** [comment]
That is so strange to have so much reserved space on the top of the screen...

screen shot 2016-11-21 at 19 03 21

**Comment by just-boris** [comment]
Agree.
What about the second option: put full name bellow the title?

**Comment by baev** [comment]
screen shot 2016-11-21 at 19 11 15

**Comment by baev** [comment]
test this please

**Pull request merged**